### PR TITLE
feat: remove prev button from the last step of telegraf overlay

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
Closes https://github.com/influxdata/ui/issues/2751

Remove the previous button from the last step of the Telegraf Refresh UI Overlay. (The one where it shows the instructions, the final step.)

Screenshot:

Capture d’écran, le 2021-10-01 à 12 26 22 PM

### CODE REVIEW

File: src/dataLoaders/components/collectorsWizard/verify/VerifyCollectorsStep.tsx [hunk 1]
```diff
@@ -57,11 +56,15 @@ export class VerifyCollectorStep extends PureComponent<Props> {
             configID={telegrafConfigID}
           />
         </DapperScrollbars>
-        <OnboardingButtons
-          onClickBack={onDecrementCurrentStepIndex}
-          nextButtonText="Finish"
-          className="data-loading--button-container"
-        />
```
Comment (ChitlangeSahas) [inline src/dataLoaders/components/collectorsWizard/verify/VerifyCollectorsStep.tsx:64]: We didn't need the collection of buttons anymore so i just removed this and added a simple button to the footer.

File: src/dataLoaders/components/collectorsWizard/verify/VerifyCollectorsStep.tsx [hunk 2]
```diff
@@ -57,11 +56,15 @@ export class VerifyCollectorStep extends PureComponent<Props> {
...
+
+        <Overlay.Footer className="data-loading--button-container">
+          <Button
+            color={ComponentColor.Primary}
+            text="Finish"
+            type={ButtonType.Submit}
+            testID="next"
+          />
+        </Overlay.Footer>
```
Comment (TCL735) [inline src/dataLoaders/components/collectorsWizard/verify/VerifyCollectorsStep.tsx:67]: Unfortunately, we cannot put an `Overlay.Footer` here. This component is child of a `Overlay.Body` - see https://github.com/influxdata/ui/blob/master/src/writeData/components/PluginCreateConfigurationWizard.tsx#L143-L152 and https://github.com/influxdata/ui/blob/master/src/writeData/components/PluginAddToExistingConfiguration/Wizard.tsx#L133-L141

Per @daviddkkim the footer should not be inside the body.

There's two choices here. Add the button to the Footer component in two places. Or, just use a `<Button>` here with possibly some CSS to position it correctly.

**Review by TCL735 — CHANGES_REQUESTED** [review]

**Comment by TCL735** [comment]
You will also need to rebase with master to get the e2e tests passing.

File: src/dataLoaders/components/collectorsWizard/verify/VerifyCollectorsStep.tsx (hunk 2 above)
Comment (ChitlangeSahas) [reply src/dataLoaders/components/collectorsWizard/verify/VerifyCollectorsStep.tsx]: This makes sense yes! I just fixed it, used a flexbox instead seems to be working the same!

**Review by TCL735 — APPROVED** [review]

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 65 | :yellow_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2758)

**Review by TCL735 — APPROVED** [review]

**Pull request merged**